go/src/socialapi/workers/countly/api/api.go

```diff
@@ -0,0 +1,123 @@
+package api
+
+import (
+	"koding/db/mongodb/modelhelper"
+	"koding/tools/utils"
+	"net/http"
+	"net/url"
+	"socialapi/config"
+	"socialapi/models"
+	"socialapi/workers/common/response"
+	"socialapi/workers/countly/client"
```

Comment:
i dont do the imports, that is what goimport does, and i dont want to manage them by hand that is machine's job ;)

we may need to fork and customize it a little bit. wdyt?